### DISCUSSION TITLE
View mutation details in DevTools 6.0.12?

### DISCUSSION THREAD

**Discussion by plweil** [vuejs/pinia#1120] [Help and Questions]
I just got a look at how my Pinia stores are supported in DevTools. Are we supposed to see more detailed mutation info, similar to what we could see with Vuex? All I'm seeing is the name of the relevant store. If that's all we're supposed to see, that's disappointing and not very useful. I'm still using Vue 2 for now.

![Screen Shot 2022-03-04 at 2 55 07 PM](https://user-images.githubusercontent.com/1033771/156840341-023cc6c9-04d2-4993-8fbe-a23f893d5c12.png)

**Comment by dmcknight26** [comment]
I am seeing the same thing, whether I manually update a property (`userStore.foo = 42`) or use patch (`userStore.$patch({ foo: 42 })`). Is there no way to see the actual details of the mutation in devtools?
